### PR TITLE
fix an issue that may cause deployment fails

### PR DESCRIPTION
**Pull request by huihzhao**

### CODE REVIEW

File: consumers/bas.md [hunk 1]
```diff
@@ -7,35 +7,51 @@ If, as is highly likely, your BAS chain doesn't already have a deployment of Pyt
...
     # Testnet Pyth
-    # PYTH_TO_WORMHOLE_EMITTER=0xf346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0
+    PYTH_TO_WORMHOLE_EMITTER=0xf346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0
     # Mainnet Pyth
     # PYTH_TO_WORMHOLE_EMITTER=0x6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25
     ```
+    
+ - Now, under the pyth2wormhole/ethereum directory, install dependencies.
+
+    ```shel
```
Comment (ali-bahjati) [inline consumers/bas.md:49]:
```suggestion
    ```shell
```

File: consumers/bas.md [hunk 2]
```diff
@@ -7,35 +7,51 @@ If, as is highly likely, your BAS chain doesn't already have a deployment of Pyt
...
+    
+    # Wormhole Core Migrations
+    INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
+    INIT_GOV_CHAIN_ID=0x1
+    INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+    
     # Choose this value based on your network type (testnet, mainnet)
     # Testnet Pyth
-    # PYTH_TO_WORMHOLE_EMITTER=0xf346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0
+    PYTH_TO_WORMHOLE_EMITTER=0xf346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0
```
Comment (ali-bahjati) [inline consumers/bas.md:42]: Can you please add wormhole env variable here and also specify mainnet ones?

like so:

```
    # Choose this value based on your network type (testnet, mainnet)
    # Testnet Pyth
    PYTH_TO_WORMHOLE_EMITTER=0xf346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0
    INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
    INIT_GOV_CHAIN_ID=0x1
    INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
    # Mainnet Pyth
    # PYTH_TO_WORMHOLE_EMITTER=0x6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25
    # INIT_SIGNERS=["0x58CC3AE5C097b213cE3c81979e1B9f9570746AA5"]
    # INIT_GOV_CHAIN_ID=0x1
    # INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
```

**Review by ali-bahjati — CHANGES_REQUESTED** [review]
Thank you so much ser! left some minor comments. I'll approve and merge after addressing them.

File: consumers/bas.md (hunk 2 above)
Comment (huihzhao) [reply consumers/bas.md:49]: done

File: consumers/bas.md (hunk 1 above)
Comment (huihzhao) [reply consumers/bas.md]: > 

fixed

**Review by huihzhao — COMMENTED** [review]
Done

**Comment by huihzhao** [comment]
Please review again and merge the request. Thanks

**Review by ali-bahjati — APPROVED** [review]
Nice! tyvm 🎉

**Pull request merged**